lib/src/number-input/NumberInputContext.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+interface INumberInputContext {
```

Comment:
I think we should use a type instead of an interface, as we are currently only using types.

Reply:
Done.